File: java/1006-media_2.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  public static void main(String[] args) throws IOException {
    double a, b, c, media;

    Scanner scanner = new Scanner(System.in);
    a = scanner.nextDouble();
    b = scanner.nextDouble();
    c = scanner.nextDouble();
    scanner.close();

    media = ((a*2) + (b*3) + (c*5)) / 10;
    System.out.printf("MEDIA = %.1f\n", media);
  }
}

File: kotlin/1006-media_2.kt
import java.util.*

fun main(args: Array<String>) {
  val A = readLine()!!.toDouble()
  val B = readLine()!!.toDouble()
  val C = readLine()!!.toDouble()
  val MEDIA = ((A*2) + (B*3) + (C*5)) / 10

  println("MEDIA = " + "%.1f".format(MEDIA))
}
